extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ClassConfigPropertiesUtil.java

```diff
@@ -167,7 +170,8 @@ static boolean addProducerMethodForClassConfigProperties(ClassLoader classLoader
                 configObjectClassStr, produceMethodParameterTypes)) {
             methodCreator.addAnnotation(Produces.class);
 
-            ResultHandle configObject = populateConfigObject(classLoader, configPropertiesClassInfo, prefixStr, methodCreator,
+            ResultHandle configObject = populateConfigObject(classLoader, configPropertiesClassInfo, prefixStr, namingStrategy,
+                    methodCreator,
```

Comment:
Small thing, but maybe move this inline with `applicationIndex, configProperties);`